roles/openshift_metrics/README.md

```diff
@@ -16,6 +16,7 @@ From this role:
 | Name                                            | Default value         |                                                             |
 |-------------------------------------------------|-----------------------|-------------------------------------------------------------|
 | openshift_hosted_metrics_deploy                 | `False`               | If metrics should be deployed                               |
+| hawkular_metrics_hostname                       | `hawkular-metrics.<default_subdomain>` | Metrics hostanme                       |
```

Comment:
a. The mark down is parsed correctly, I did not want to edit the other lines and move the `|` mark ...
b. Added a new commit to fix the align :-)